lib/Models/getToken.js

```diff
@@ -31,7 +31,7 @@ function getToken(terria, tokenUrl, url) {
 
       return tokenResponse.token;
     })
-    .otherwise(() => {
+    .catch(() => {
```

Comment:
Thanks! Not sure how that went through, maybe I forgot to commit the file.

Reply:
I think it was needed because of new changes on `mobx` that I merged in.